src/main/thrift/parquet.thrift

```diff
@@ -191,6 +191,64 @@ enum FieldRepetitionType {
   REPEATED = 2;
 }
 
+/**
+  * A histogram of repetition and definition levels for either a page or column chunk. 
+  *
+  * This is useful for:
+  *   1. Estimating the size of the data when materialized in memory 
```

Comment:
(2) can be easily done with these statistics, but I wonder how can we estimate the size of data with rep-def histogram...Is there any formulas?

Reply:
Oh I got it, since we can deduce that the size of parent layer...Thats great